shared/utils.js

```diff
@@ -70,7 +70,7 @@ function setLogLevel(logLevel) {
  */
 function getVersionNumberFromString(versionString) {
 	// Only supporting major/minor version checking at this point.
-	var versionRegex = /^(\d+)(\.(\d+))?.*$/;
+	var versionRegex = /^[^\d]*(\d+)(\.(\d+))?.*$/;
```

Comment:
@wmathurin My regex skills are poor.  Does this look fine?  Just trying to skip leading non-numbers because I have seen Cordova print strings.

Reply:
Looks good.